### PR TITLE
Make deathclaws great again

### PR DESCRIPTION
**Pull request by oprimeolt**
## Pre-Merge Checklist
- [x] You tested this on a local server.
- [x] This code did not runtime during testing.
- [x] You documented all of your changes.

## Changelog
:cl: Oprime
balance: deathclaw health
/:cl:

### CODE REVIEW

File: code/modules/mob/living/simple_animal/hostile/f13/deathclaw.dm [hunk 1]
```diff
@@ -141,8 +141,8 @@
 	desc = "A massive, reptilian creature with powerful muscles, razor-sharp claws, and aggression to match. This one is an angry mother."
 	gender = FEMALE
 	mob_armor = ARMOR_VALUE_DEATHCLAW_MOTHER
-	maxHealth = 300
-	health = 300
+	maxHealth = 1500
```
Comment (xTrainx) [inline code/modules/mob/living/simple_animal/hostile/f13/deathclaw.dm:144]: seems a bit excesive, let's see how it rolls out

File: code/modules/mob/living/simple_animal/hostile/f13/deathclaw.dm [hunk 2]
```diff
@@ -167,8 +167,8 @@
 	name = "legendary deathclaw"
 	desc = "A massive, reptilian creature with powerful muscles, razor-sharp claws, and aggression to match. This one is a legendary enemy."
 	mob_armor = ARMOR_VALUE_DEATHCLAW_MOTHER
-	maxHealth = 400
-	health = 400
+	maxHealth = 2400
```
Comment (xTrainx) [inline code/modules/mob/living/simple_animal/hostile/f13/deathclaw.dm:170]: woah

**Comment by xTrainx** [comment]
PA claw is untouched

**Comment by Aurrain** [comment]
Gave the PA claw a health value that seems in-line with the rest so this PR could be done.

**Pull request merged**